src/Components/Button.js

```diff
@@ -1,16 +1,32 @@
 import React from "react";
 import PropTypes from "prop-types";
 const Button = (props) => {
-  const { name } = props;
+  
+  const { name, color, wide } = props;
+
   return (
-    <button type="button" name={name} id={name} className="button">
+    <button
+      type="button"
+      style={{ backgroundColor: color, width: wide }}
+      name={name}
+      id={name}
+      className="button"
+    >
       {name}
     </button>
   );
 };
 
 Button.propTypes = {
   name: PropTypes.string,
+  color: PropTypes.string,
+  wide: PropTypes.string,
+};
```

Comment:
- [ ] The prop `name` should be declared as a required prop in the propTypes declaration since a default value was not provided for it. Eslint is going to start complaining about this after being properly set up.
```JS
Button.propTypes = {
  name: PropTypes.string.isRequired,
  color: PropTypes.string,
  wide: PropTypes.string,
};
```